src/brevitas_examples/imagenet_classification/qat/README.md

```diff
@@ -5,12 +5,12 @@ and by no means a direct mapping to hardware should be assumed.
 
 Below in the table is a list of example pretrained models made available for reference.
 
-| Name         | Cfg                   | Scaling Type               | First layer weights | Weights | Activations | Avg pool | Top1  | Top5  | Pretrained model                                                                                | Retrained from                                                |
```

Comment:
Any reason why you also removed Top-5?

Reply:
We don't compute top5 accuracy anymore because:
- It is not as relevant as the Top 1
- I went down the rabbit hole of removing/reducing variability while performing 2 runs with the same seed, and this has led to removing the computation of top5 accuracy. Since we don't generate it while running the script, I thought I would also remove it from the results as it could be confusing.

I am happy to restore it in the readme, but less inclined to restore its computation.